router/internal/router/config.go

```diff
@@ -229,7 +234,7 @@ func (config *Config) loadListener(rootConfigMap map[interface{}]interface{}) er
 		}
 	}
 
-	if (edgeBinding == nil) && (edgeWssBinding == nil) {
+	if (edgeBinding == nil) && (edgeWsBinding == nil) {
 		return errors.New("required binding [edge] not found in [listeners]")
```

Comment:
I would augment this message to say

`required binding [edge] not found in [listeners], at least one edge binding is required`

Reply:
done